app/renderer/actions/actionTypes.ts

```diff
@@ -1,5 +1,8 @@
 import { IActionWithPayload } from "app/renderer/actions/helpers"
 import { Wallet } from "app/common/runtimeTypes/storage/wallets"
+import { ComputedTransaction } from "app/renderer/prefilledTransactions"
 
 // Types for the actions that arrive at the wallet reducer
-export type SaveWalletAction = IActionWithPayload<Wallet>
\ No newline at end of file
+export type SaveWalletAction = IActionWithPayload<Wallet>
+
+export type SaveTransactionsAction = IActionWithPayload<ComputedTransaction>
```

Comment:
I think it's not a single transaction that should be stored in redux, so it should be `IActionWithPayload<ComputedTransactions>`